app/views/users/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= form_for(@user)  do |f| %>
+  <% if @user.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@user.errors.count, "error") %> prohibited this user from being saved:</h2>
+
+      <ul>
+      <% @user.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+  <p>
+ <%= f.text_field :username, :value => 'John Doe' %>
+  <%= f.text_field :email, :value => "johnny123@gmail.com"%>
+  <%= f.text_field :password, placeholder: "password" %>
+  </p>
+
+  <p>
+  <%= f.submit "Submit" %>
+  </p>
+   <p> Last added Users</p>
+
+    <ul id="users">
+  <% @users.each do |user| %>
+    <li>
+        <%= 'Username = ' + user.username  %><br>
+        <%= 'Email = ' + user.email  %><br>     
+          <%= link_to "Edit User", edit_user_path(user.id) %>
+          <br> <br> 
+    </li>
+    <% end %>
+</ul>
+<% end %>
```

Comment:
- **_As mentioned in the previous review: There should be an Html form which should be commented out_**

- **_As mentioned in the previous review: There should be a form_tag which should be commented out_**